contracts-v0.2/src/Aggregator.sol

```diff
@@ -173,12 +152,7 @@ contract Aggregator is Ownable, IAggregator, ITypeAndVersion {
             addOracle(_added[i]);
         }
 
-        updateFutureRounds(
-            _minSubmissionCount,
-            _maxSubmissionCount,
-            _restartDelay,
-            timeout
-        );
+        updateFutureRounds(_minSubmissionCount, _maxSubmissionCount, _restartDelay, timeout);
```

Comment:
We should not modify the the code that is not meant to be changed. Please roll back all the formatting changes. We should not mix formatting PRs with functionality changes.